starcheck/calc_ccd_temps.py

```diff
@@ -118,16 +119,19 @@ def get_ccd_temps(oflsdir, outdir='out',
     if not os.path.exists(outdir):
         os.mkdir(outdir)
 
+    run_start_time = DateTime(run_start_time)
     config_logging(outdir, verbose)
 
     # Store info relevant to processing for use in outputs
     proc = {'run_user': os.environ['USER'],
-            'run_time': time.ctime(),
+            'actual_run_time': time.ctime(),
```

Comment:
If you are changing the name, how about `execution_time`?

Reply:
OK.